src/modules/refreshScores.js

```diff
@@ -0,0 +1,9 @@
+import fetchScores from './fetchScores.js';
+
+const refreshScores = async () => {
+  document.getElementById('refresh-btn').addEventListener('click', () => {
+    fetchScores();
+  });
+};
+
```

Comment:
- [ ] At the moment this function is redundant because the scores are posted directly to the UI, kindly refactor your submitForm.js in such a way that scores are firstly submitted to the API; then, the refresh button will retrieve the score from the API and display it in the UI. 👍 

<img width="635" alt="Untitled" src="https://github.com/Titus-Kiplagat/leaderboard/assets/103386183/fe171082-0a0f-44b5-ba82-f2f27b9c7fda">